NuGet.Config

```diff
@@ -1,9 +1,13 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <configuration>
   <packageSources>
+    <clear/>
```

Comment:
Please do not recommit the change the same as main branch, which will cause conflict when merge back to master